qchem/tests/hf/test_dipole.py

```diff
@@ -0,0 +1,334 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for functions needed for computing the dipole.
+"""
+import autograd
+import pennylane as qml
+import pytest
```

Comment:
Do we worry about pylint warnings related to `third party import X should be placed before Y`?

Reply:
I do not see such a warning, where is it exactly?

Reply:
While running pylint locally, on lines with  `import pytest` I see the following: 
`C0411: third party import "import pytest" should be placed before "import pennylane as qml" (wrong-import-order)`

Reply:
Fixed, thanks.